packages/core/admin/admin/src/content-manager/components/AttributeFilter/hooks/useAllowedAttributes.js

```diff
@@ -14,29 +14,39 @@ const useAllowedAttributes = (contentType, slug) => {
     },
   ]);
 
+  const canReadAdminUsers =
+    findMatchingPermissions(allPermissions, [
+      {
+        action: 'admin::users.read',
+        subject: null,
+      },
+    ]).length > 0;
+
   const readPermissionForAttr = get(readPermissionsForSlug, ['0', 'properties', 'fields'], []);
   const attributesArray = Object.keys(get(contentType, ['attributes']), {});
-  const allowedAttributes = attributesArray
-    .filter((attr) => {
-      const current = get(contentType, ['attributes', attr], {});
+  const allowedAttributes = attributesArray.filter((attr) => {
+    const current = get(contentType, ['attributes', attr], {});
+
+    if (!current.type) {
+      return false;
+    }
 
-      if (!current.type) {
-        return false;
-      }
+    if (NOT_ALLOWED_FILTERS.includes(current.type)) {
+      return false;
+    }
 
-      if (NOT_ALLOWED_FILTERS.includes(current.type)) {
-        return false;
-      }
+    if (!readPermissionForAttr.includes(attr) && attr !== 'id' && !TIMESTAMPS.includes(attr)) {
+      return false;
+    }
 
-      if (!readPermissionForAttr.includes(attr) && attr !== 'id' && !TIMESTAMPS.includes(attr)) {
-        return false;
-      }
+    return true;
+  });
 
-      return true;
-    })
-    .sort();
+  if (canReadAdminUsers) {
+    allowedAttributes.push('createdBy', 'updatedBy');
+  }
 
-  return allowedAttributes;
+  return allowedAttributes.sort();
```

Comment:
I don't think this would work as we anticipate with different languages?

Reply:
I more meant if any of the attributes are in a different language. We had problems in the CTB & CM where they were incorrectly ordered due to language nuances e.g in Russian. Assuming you the filter prints the attribute names as is we should probably use the same method of sorting here. I'm pretty sure the hooks is called useCollator if you want to search how it's done elsewhere 🤙🏼